src/main/java/de/picturesafe/search/parameter/aggregation/DefaultAggregation.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 picturesafe media/data/bank GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.picturesafe.search.parameter.aggregation;
+
+import de.picturesafe.search.util.logging.CustomJsonToStringStyle;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+/**
```

Comment:
Maybe better: ?

Aggregation with default parameters.
The type of the aggregation will be determined by the field type.

Reply:
OK